engine-types/src/types.rs

```diff
@@ -107,6 +109,13 @@ impl Add for Wei {
     }
 }
 
+/// Type casting from Wei compatible Borsh-encoded raw value into the Wei value, to attach an ETH balance to the transaction
+impl From<WeiU256> for Wei {
+    fn from(value: WeiU256) -> Self {
+        Wei(value.into())
```

Comment:
I prefer calling `U256::from_big_endian` directly, just to be clear about what the endianness of the encoding.

Reply:
Addressed notes and remarks here, made a commit with corrections and improvements:
https://github.com/aurora-is-near/aurora-engine/pull/351#issuecomment-969075880